sdformat_to_mjcf/sdformat_to_mjcf/converters/joint.py

```diff
@@ -63,10 +65,13 @@ def add_joint(body, joint):
     :return: The newly created MJCF joint.
     :rtype: mjcf.Element
     """
-    if joint is None:
+    if isinstance(joint, FreeJoint):
         return body.add("freejoint")
 
-    pose = su.graph_resolver.resolve_pose(joint.semantic_pose())
+    if isinstance(joint, StaticFixedJoint):
+        pose = Pose3d()
```

Comment:
it's ok that this is an empty pose because the pose of a static fixed joint doesn't matter right? it could be located anywhere and would still have the same effect. if that's correct and since it took me a minute to realize this, perhaps we could add a comment that effect?

Reply:
Yup, added a comment in https://github.com/gazebosim/gz-mujoco/pull/53/commits/e21095ccd0333db44ad2f5eb04d2f04b72044932.